### DISCUSSION TITLE
How to get silhouette of DXF

### DISCUSSION THREAD

**Discussion by git-n-pissed** [mozman/ezdxf#660] [Q&A]
The DXF below was generated from a revolved profile in SolidWorks:

![image](https://user-images.githubusercontent.com/52613263/160184221-484b180e-e266-4c06-843e-1bb1a7da69c2.png)

I would like to have ezdxf remove or ignore all portions of lines which are not part of the silhouette.  The result would be something like this:

![image](https://user-images.githubusercontent.com/52613263/160184408-ca0c1f5b-4d16-4ec5-aef8-5986bd783dcf.png)

Is there a good way to accomplish this with ezdxf?

**Comment by mozman** [comment]
No, `ezdxf` is an interface to the DXF file format not a CAD application.
